pos-ui-extension/shopify.extension.toml.liquid

```diff
@@ -1,8 +1,27 @@
-type = "pos_ui_extension"
+# The version of APIs your extension will receive. Learn more:
+# https://shopify.dev/docs/api/usage/versioning
+api_version = "2024-04"
+
+[[extensions]]
+type = "ui_extension"
 name = "{{ name }}"
-description = "{{ name }}"
+handle = "{{ handle }}"
+description = "A {{ flavor }} POS UI extension"
```

Comment:
Whats `flavor`

Reply:
Also what is `npm run shopify app generate extension -- --clone-url https://github.com/Shopify/extensions-templates#njo/pos-2024-04` exactly doing? Its migrating your existing extension over to these templates?

Reply:
Thanks for the questions! 

`flavor` refers to the selection of `JavaScript`, `Typescript`, `React`, `React Typescript`

the `--clone-url` option specifies where the CLI should generate the extensions from. It's specifically creating a new extension using this very branch.